iree-jax/benchmark/benchmark_all.sh

```diff
@@ -62,12 +62,16 @@ fi
 rm "${OUTPUT_PATH}"
 echo "{\"trigger\": { \"timestamp\": \"$(date +'%s')\" }, \"benchmarks\": []}" > "${OUTPUT_PATH}"
 
+CACHE_DIR="$(pwd)/.cache/oobi/models"
```

Comment:
Maybe make this configurable and default to `/tmp`
```suggestion
CACHE_DIR="${OOBI_CACHE_DIR:-/tmp/oobi/cache/models}"
```

Reply:
Done